src/main/resources/db/migration/V1_314__OFFENDER_BELIEFS.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE OFFENDER_BELIEFS (
+  OFFENDER_BOOK_ID NUMBER(10,0) NOT NULL,
+  ROOT_OFFENDER_ID NUMBER(10,0) NOT NULL,
+  BELIEF_ID NUMBER(10,0) NOT NULL,
+  BELIEF_CODE VARCHAR2(12) NOT NULL,
+  EFFECTIVE_DATE DATE,
+  END_DATE DATE,
+  CHANGE_REASON VARCHAR2(1) DEFAULT 'N',
+  COMMENTS VARCHAR2(4000),
```

Comment:
What a strange design. Does NOMIS force you to enter some comments if change_reason='Y'?